### DISCUSSION TITLE
Looking for the HD200 flash file

### DISCUSSION THREAD

**Discussion by weston1968** [OpenSageTV/Community#63] [SageTV Extenders (HD300,HD200)]
I have the HD300 bin file for reinstalling the firmware and have used it a few times over the years.  Just got an HD200 and am looking for the flash files.

**Comment by JustFred51** [comment]
[HD200 - v20100909 0 final release.zip](https://github.com/OpenSageTV/Community/files/12339985/HD200.-.v20100909.0.final.release.zip)
Here ya go...

**Comment by weston1968** [comment]
Much Thanks

On Mon, Aug 14, 2023 at 6:31 PM Keith Fischer ***@***.***>
wrote:

> HD200 - v20100909 0 final release.zip
> <https://github.com/OpenSageTV/Community/files/12339985/HD200.-.v20100909.0.final.release.zip>
> Here ya go...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenSageTV/Community/discussions/63#discussioncomment-6725644>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG23RAB2ZGAL2SAZ4NL7CSDXVKRMTANCNFSM6AAAAAA3QH3YMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
"The only thing necessary for the triumph of evil, is for good men to do
nothing."  Edmund Burke  1729 – 1797
"Name a single time in history when the group fighting to censor speech and
ban books were the good guys".  Kevin Sorbo
"Those who can make you believe absurdities; can make you commit
atrocities." Voltaire 1694 - 1728
